inc/post-type-meta-box/profile-details-page-meta-box.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * ------------------ CONTENTS ---------------------------
+ * User: mdiaconita
+ * 1. Define the inputs and their label
+ * 2. Textarea arguments
+ * 3. Generate input fields
+ * 4. Loop through array and create a get_post_meta for each element
+ * 5. Field names
+ * 6. HTML start table
+ * 7. HTML end table
+ * 8. Generate meta box for user profiles
+ * 9. Save user profile meta box
+ */
+
+function profile_define_input_type_label_meta()
+{
+    return $arr_inputs = array(
+        'user_profile_position' => array('Title detail', 'text'),
+        'user_profile_contact' => array('Email address', 'text'),
+        /** --------------------------------------------------------------- */
+        'user_profile_extra_headline' => array('Profile banner 1 title', 'text'),
+        'user_profile_extra' => array('Profile banner 1 content', 'textarea'),
+        /** --------------------------------------------------------------- */
+        'user_profile_extra_two_headline' => array('Profile banner 2 title', 'text'),
+        'user_profile_extra_two' => array('Profile banner 2 content', 'textarea'),
+        /** --------------------------------------------------------------- */
+        'user_profile_extra_three_headline' => array('Profile banner 3 title', 'text'),
+        'user_profile_extra_three' => array('Profile banner 3 content', 'textarea'),
+        /** --------------------------------------------------------------- */
+        'user_profile_extra_four_headline' => array('Profile banner 4 title', 'text'),
+        'user_profile_extra_four' => array('Profile banner 4 content', 'textarea')
+    );
+}
+
+/**
+ * ------------------ CONTENTS ---------------------------
+ * User: mdiaconita
+ * 2. Textarea arguments
+ */
+function profile_textarea_arguments()
+{
+    return $args = array(
+        'media_buttons' => false,
+        'textarea_rows' => 3,
+        'tinymce' => false,
+        'quicktags' => array('buttons' => 'strong,em,ul,ol,li,link'),
+        'wpautop' => false
+    );
+}
+
+/**
+ * 3. Generate input fields
+ * @param $input_type
+ * @param $meta
+ * @param $arg
+ * @return string
+ */
+function profile_generate_input($input_type, $meta, $arg)
+{
+    if ($input_type == 'text' || $input_type == 'tel' || $input_type == 'email') {
+        echo '<tr>',
```

Comment:
Nothing wrong with the code, but i like to suggest using
`sprintf()`

Just to make it a little bit more readable.